packages/chrome/src/styled/subnav/StyledSubNavItem.tsx

```diff
@@ -7,42 +7,44 @@
 
 import styled, { ThemeProps, DefaultTheme, css } from 'styled-components';
 import { rgba } from 'polished';
-import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { retrieveComponentStyles, DEFAULT_THEME, focusStyles } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'chrome.subnav_item';
 
 const colorStyles = (props: IStyledSubNavItemProps) => {
-  const BLACK = props.theme.palette.black as string;
-  const WHITE = props.theme.palette.white as string;
+  const { theme, isLight, isCurrent } = props;
+  const DARK = theme.palette.black as string;
+  const LIGHT = theme.palette.white as string;
   let currentColor;
   let hoverColor;
 
-  if (props.isCurrent) {
-    if (props.isLight) {
-      currentColor = rgba(BLACK, 0.1);
+  if (isCurrent) {
+    if (isLight) {
+      currentColor = rgba(DARK, 0.1);
     } else {
-      currentColor = rgba(WHITE, 0.1);
+      currentColor = rgba(LIGHT, 0.1);
     }
   } else {
-    hoverColor = rgba(props.isLight ? WHITE : BLACK, 0.1);
+    hoverColor = rgba(isLight ? LIGHT : DARK, 0.1);
   }
 
-  const activeColor = rgba(props.isLight ? BLACK : WHITE, 0.03);
-  const focusColor = rgba(props.isLight ? BLACK : WHITE, 0.2);
+  const activeColor = rgba(isLight ? DARK : LIGHT, 0.03);
 
   return css`
-    opacity: ${props.isCurrent ? '1' : '0.6'};
+    opacity: ${isCurrent ? '1' : '0.6'};
     background-color: ${currentColor};
 
     &:hover {
       opacity: 1;
       background-color: ${hoverColor};
     }
 
-    &[data-garden-focus-visible] {
-      opacity: 1;
-      box-shadow: ${props.theme.shadows.md(focusColor)};
-    }
+    ${focusStyles({
+      theme,
+      hue: isLight ? DARK : LIGHT,
```

Comment:
Focus styling for collapsible subnav items with children appears to be broken.

Reply:
Could you explain which part appears to be broken?

Reply:
Select a light `hue` and tab through the `SubNav`.

![Kapture 2023-06-01 at 15 14 32](https://github.com/zendeskgarden/react-components/assets/143773/0437efea-879b-40f4-a640-7fbb4b3171d9)

Reply:
Added #1558 to address this!